### PR TITLE
[FIX] l10n_cl_toponym

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Review by nelsonramirezs — APPROVED** [review]

**Review by nelsonramirezs — APPROVED** [review]

**Comment by dreispt** [comment]
Looks like a duplicate of #70 
Can you review that one please?

**Comment by dreispt** [comment]
This was fixed in #70, right?

**Comment by max3903** [comment]
@dreispt This one adds the `@api.multi`.

File: l10n_cl_toponym/models/res_partner.py [hunk 1]
```diff
@@ -40,30 +40,33 @@ def _vat_unique(self):
 
     @api.onchange('vat')
     def onchange_vat(self):
-        vat = self.vat
+        if self.vat:
+            vat = self.vat
         if vat:
```
Comment (bodedra) [inline l10n_cl_toponym/models/res_partner.py:44]: @max3903 This will fail if we don't have vat. (LocalVariableNotFound)

**Review by bodedra — APPROVED** [review]
Code review. LGTM :+1:

**Review by dreispt — APPROVED** [review]

**Pull request merged**